src/Program.spec.ts

```diff
@@ -2278,7 +2278,7 @@ describe('Program', () => {
             expectTypeToBe(table.getSymbolType('ifDeviceInfo', opts).getMemberType('GetRandomUUID', rtOpts), TypedFunctionType);
 
             expectTypeToBe(table.getSymbolType('ifSGNodeField', opts), InterfaceType);
-            expectTypeToBe(table.getSymbolType('ifSGNodeField', opts).getMemberType('observeField', rtOpts), TypedFunctionType);
+            expectTypeToBe(table.getSymbolType('ifSGNodeField', opts).getMemberType('addFields', rtOpts), TypedFunctionType);
```

Comment:
Needed to change this because there `observeField` is overloaded, so it is actually a UnionType... see: https://github.com/rokucommunity/brighterscript/issues/926